src/test/scala/org/clulab/habitus/variables/TestEntitiesNorms.scala

```diff
@@ -0,0 +1,307 @@
+    VariableTest(
+      passingTest,
+      "sent20",
+      "In the SRV, production areas are typically larger in the dry season, which brings fewer problems with pests and birds (Tanaka et al., 2015; USDA-GAIN, 2018).",
+      Seq(
+        "Date" -> Seq(("2015", "2015-XX-XX"), ("2018","2018-XX-XX"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent21",
+      "Average yields in SRV theoretically range between 5.0 and 6.0 t ha-1 in the rainy season and between 6.5 and 7.5 t ha-1 in the dry season (SAED, 2019; USDA-GAIN, 2021)",
+      Seq(
+        "Date" -> Seq(("2021", "2021-XX-XX"), ("2019", "2019-XX-XX")),
+        "Quantity" -> Seq(("between 6.5 and 7.5 t ha-1", "6.5 -- 7.5 t/ha"), ("between 5.0 and 6.0 t ha-1", "5.0 -- 6.0 t/ha"))
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent22",
+      "seeds of the rice variety Sahel 108 are sown, a short- cycle variety (around 125 days)",
+      Seq(
+        "Quantity" -> Seq(("125 days", "125.0 d")),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent23",
+      "Broadcast seeding is carried out by hand on irrigated plots with a 2-5 cm depth sheet of water",
+      Seq(
+        "Quantity" -> Seq(("2-5 cm", "2.0 -- 5.0 cm")),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent24",
+      "average yields for the two seasons assessed of 4832 kg ha− 1 and 7425 kg ha− 1 for the areas under CONV and INT management, respectively.",
+      Seq(
+        "Quantity" -> Seq(("4832 kg", "4832.0 kg"), ("7425 kg", "7425.0 kg"))
+      )
+    ),
+    VariableTest(
+      failingTest,
+      "sent25",
+      "In plots receiving fertilizer, DAP was applied basally (19.3 and 21.5 kg N and P ha−1 ), and three urea splits were broadcasted into 1–5 cm of water (101.3 kg N ha−1 ; 40% at early-tillering, 40% at pan- icle initiation, and 20% at heading)",
+      Seq(
+        // FIXME; `19.3 and 21.5 kg` not extracted fully (without 19.3). Also some wrong Quantity extractions. Discuss with Masha.
+        "Quantity" -> Seq(("101.3 kg", "101.3 kg"), ("1–5 cm", "1.0 -- 5.0 cm"), ("21.5 kg", "21.5 kg")),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent26",
+      "Transplanting for both management systems took place on 7 (2007 and 2008 wet season) and 25 (2009 wet season) August and on 19 March (2008 and 2009 dry season).",
+      Seq(
+        "Date" -> Seq(("2007", "2007-XX-XX"), ("2008", "2008-XX-XX"), ("2009", "2009-XX-XX"),
+                        ("August", "XXXX-08-XX"), ("19 March", "XXXX-03-19"), ("2008", "2008-XX-XX"), ("2009", "2009-XX-XX")),
+      )
+    ),
+    VariableTest(
+      failingTest,
+      "sent41",
+      "das days after sowing, Fert fertilizer treatment, with F1: recommended dose (80 kg N ha-1), i.e., 200 kg ha-1 NPK (15.15.15) at sowing + 100 kg ha-1 urea at 20 das + 50 kg ha-1 urea at 50 das. F2: F1/4 (20 kg N ha-1);",
+      Seq(
+        // FIXME; Extraction of quantities with `+` signs here.
```

Comment:
Let's discuss in the lab meeting next week! *this should not stop the merge into main, fine to keep this failing for now